service-tests/twitter.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const {
+  isMetric
+} = require('./helpers/validators');
+
+const t = new ServiceTester({ id: 'twitter', title: 'Twitter' });
+module.exports = t;
+
+t.create('Followers')
+  .get('/follow/shields_io.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: 'Follow @shields_io',
+    value: isMetric
+  }));
+
+t.create('Followers - Custom Label')
+  .get('/follow/shields_io.json?label=Follow')
+  .expectJSONTypes(Joi.object().keys({
+    name: 'Follow',
+    value: isMetric
+  }));
+
+t.create('Invalid Username Specified')
+  .get('/follow/invalidusernamethatshouldnotexist.json?label=Follow')
+  .expectJSONTypes(Joi.object().keys({
+    name: 'Follow',
+    value: 'invalid user'
+  }));
+
+t.create('No connection')
+  .get('/follow/shields_io.json?label=Follow')
+  .networkOff()
+  .expectJSON({ name: 'Follow', value: 'inaccessible' });
+
+t.create('URL')
+  .get('/url/https/shields.io.json')
+  .expectJSON({ name: 'tweet', value: '' });
```

Comment:
This is a confusing badge URL!

Reply:
Yeah, Don't think it would be a very often used badge,
As it would need to be included as an `<object>` for the link to actually work.
Otherwise its the same as using ![](https://img.shields.io/:tweet--grey.svg?style=social&logo=twitter) `https://img.shields.io/:tweet--grey.svg?style=social&logo=twitter`

Will be good once we can get #966 implemented to see how often certain badges are being used.